readme/customization.md

```diff
@@ -101,7 +101,7 @@ The valid colors are the [eight original 8 ANSI colors][ANSIColors]. They are `b
 
 ## Key Bindings
 
-Most keys can be changed to any printable character or supported special character. It is possible to provide conflicting bindings, which will result in undefined behaviour. The `inputConfirmYes` binding has a special behaviour in that it responds to both the uppercase and lowercase letter of the value set, if the variant exist.
+Most keys can be changed to any printable character or supported special character. It is possible to provide conflicting bindings, which will result in undefined behavior. The `inputConfirmYes` binding has a special behavior in that it responds to both the uppercase and lowercase letter of the value set, if the variant exist.
```

Comment:
This is technically the Canadian (and UK) way of spelling behaviour, however, I've already used the American spelling of Colour, so this makes sense to change.

Reply:
I know, this one is open to debate.

I would say the English way would be colour or behaviour.

But that said, in code software, the US English often defines the norm.

I mean CSS refers to color, not colour.

I let you tell me if you want me to restore the behavior, I don't mind as I already have to fix Homebrew

Reply:
I agree, changing to the US spelling makes sense here!